Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -2022,13 +2035,65 @@ void _CGContextDrawImageRect(CGContextRef context, CGImageRef image, CGRect src,
 }
 
 /**
- @Status Stub
+ @Status Interoperable
 */
 void CGContextDrawTiledImage(CGContextRef context, CGRect rect, CGImageRef image) {
-    // TODO #1417: This can be combined with brushes (brush + tiled fill);
     NOISY_RETURN_IF_NULL(context);
-    UNIMPLEMENTED();
+    NOISY_RETURN_IF_NULL(image);
+
+    woc::unique_cf<CGImageRef> refImage{ __CGContextCreateRenderableImage(image) };
+
+    ComPtr<ID2D1Bitmap> d2dBitmap;
+    FAIL_FAST_IF_FAILED(__CreateD2DBitmapFromCGImage(context, refImage.get(), &d2dBitmap));
+
+    // |1  0 0| is the transformation matrix for flipping a rect about its Y midpoint m. (m = (y + h/2))
+    // |0 -1 0|
+    // |0 2m 1|
+    //
+    // Combined with [scale sx * sy] * [translate X, Y], that becomes:
+    // |sx     0 0|
+    // | 0   -sy 0|
+    // | x -y+2m 0|
+    // Or, the transformation matrix for drawing a flipped rect at a scale and offset.
+    D2D1_SIZE_U bitmapSize = d2dBitmap->GetPixelSize();
+    FAIL_FAST_IF(bitmapSize.width == 0);
+    FAIL_FAST_IF(bitmapSize.height == 0);
+
+    CGFloat sx = rect.size.width / bitmapSize.width;
+    CGFloat sy = rect.size.height / bitmapSize.height;
+    CGFloat m = rect.origin.y + (rect.size.height / 2.f);
+
+    CGAffineTransform transform{ sx, 0, 0, -sy, rect.origin.x, (2 * m) - rect.origin.y };
+    transform = CGAffineTransformConcat(transform, CGContextGetUserSpaceToDeviceSpaceTransform(context));
```

Comment:
I think I saw the same block of code (with identical comments) for transforms in @DHowett-MSFT's PR.  can we make a utility function out of this?